### ISSUE TITLE
Start and end of tabpanel not announced

### ISSUE THREAD

**Issue by janellenchew** [FreedomScientific/standards-support#151]
### Summary

> In JAWS 18/2018/2019, the start and end of tabpanels (role="tabpanel") are not announced. This is no indication of when you have exited the tabpanel content.

### Expected result

> JAWS should announce "tabpanel start" and "tabpanel end". This worked in JAWS 17.

### Actual result

> JAWS does not announce the start or end of tabpanel.

### Example

> https://www.w3.org/TR/wai-aria-practices/examples/tabs/tabs-2/tabs.html

### Additional Information

I have verbosity settings on high and "tabpanel" checked.

#### JAWS version and build number

JAWS 2019.1810.64

#### Operating System and version

Windows 10

#### Browser and version:

Firefox 64
Chrome 72 Beta
Edge 42

**Comment by Menelion** [comment]
Hm. Interesting, how I could miss it. Confirmed here. @stevefaulkner Thoughts, please? :)

**Comment by stevefaulkner** [comment]
@jaychewy @Menelion confirmed in chrome Version 71.0.3578.98 (Official Build) (64-bit) and JAWS 19 latest.

**Comment by stevefaulkner** [comment]
Filed internally as "Bug 106122 - Start and end of tabpanel not announced" thanks @jaychewy for filing issue!

**Comment by Rasubra8** [comment]
Any Update on this? It seems jaws version 2022 also have this issue?

**Comment by stevefaulkner** [comment]
tested with NVDA 2023.1 and chrome Version 116.0.5845.112 (Official Build) (64-bit) , start and end of tab panel **not announced**

**Comment by BrettLewisVispero** [comment]
Imported into ADO as bug 41068

**Comment by BrettLewisVispero** [comment]
Tested with JAWS 2024.
